tools/projmgr/templates/cdefault.yml

```diff
@@ -30,9 +28,9 @@ default:
         - -std=gnu11
       Link:
         - --specs=nano.specs
-        - --specs=rdimon.specs
```

Comment:
If you drop `--specs=rdimon.specs`, you likely need to add `--specs=nosys.specs` as you need some implementation of the syscalls functions.
The nosys.specs contains the default implementation (returning error) wile rdimon.specs contains a full semihosting implementation of the same syscalls.
If you have neither of them, you either need to implement the used syscalls yourself or you will get undefined symbol linker error.